src/emqtt_bench.erl

```diff
@@ -654,16 +654,25 @@ put_publish_begin_time() ->
     end,
     ok.
 
+%% @doc return new value
+inc_iner_pub_counter() ->
+    case get(iner_pub_cnt) of
+        undefined ->
+            put(iner_pub_cnt, 1), 1;
+        Val ->
+            put(iner_pub_cnt, Val+1), Val+1
+    end.
+
 next_publish(Opts) ->
-    inc_counter(pub),
```

Comment:
This change should be reverted.
The publish already completed once entering this function.

Reply:
Should not include overheads of the scheduling of next message to the global counter bump